keyboards/melgeek/z70ultra/rev1/config.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+/* USB Device descriptor parameter */
```

Comment:
Most of this (ie. the stuff that won't change in rev2) can probably go in `z70ultra/config.h` instead.